main/src/com/google/refine/model/changes/RowAdditionChange.java

```diff
@@ -102,21 +106,27 @@ static public Change load(LineNumberReader reader, Pool pool) throws Exception {
 
         String line;
         while ((line = reader.readLine()) != null && !EOC.equals(line)) {
-            int equal = line.indexOf('=');
-            CharSequence field = line.subSequence(0, equal);
-
-            if (INDEX_FIELD.contentEquals(field)) {
-                index = Integer.parseInt(line.substring(equal + 1));
-            } else if (COUNT_FIELD.contentEquals(field)) {
-                count = Integer.parseInt(line.substring(equal + 1));
-                for (int i = 0; i < count; i++) {
-                    if ((line = reader.readLine()) != null) {
-                        rows.add(Row.load(line, pool));
-                    }
-                }
-            }
+            index = processLine(reader, pool, rows, index, line);
         }
 
         return new RowAdditionChange(rows, index);
     }
+
+    private static int processLine(LineNumberReader reader, Pool pool, List<Row> rows, int index, String line) throws IOException {
+        int count;
```

Comment:
Declaring `count` where it's first used would probably be better (and the declaration in the `load()` method above looks like it's no longer used and can probably be removed).